### DISCUSSION TITLE
Dial a Ride Problem

### DISCUSSION THREAD

**Discussion by jfulem** [TimefoldAI/timefold-solver#33] [General]
Please, if timefold tool applicable for a solution to the Dial-A-Ride Problem?

**Comment by ge0ffrey** [comment]
Good idea. We need to create a quickstart for it in timefold-quickstarts.

**Comment by jfulem** [comment]
That would be fantastic!

**Comment by ge0ffrey** [comment]
The question is when will it be prioritized. That will take a while.

But we need timefold-quickstarts that show VRPTW, VRPPD and dial-a-ride
